VISION.md

```diff
@@ -0,0 +1,41 @@
+### The Danger Vision
+
+Danger is a long term project, from our current viewpoint her role is
+
+ * Provide the best API for communication in Continuous Integration.
```

Comment:
How would you describe it to a friend when meeting them? I'd probably say something like:

> You can define rules for pull requests, which will add messages and warnings right to GitHub (or any other hosted git service)